libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/AuthenticatorService.java

```diff
@@ -237,7 +247,7 @@ public Bundle getAuthToken(
             }
             final Bundle resBundle = new Bundle();
             try {
-                final TokenEndpointResponse tr = OAuth2.refreshAuthToken(HttpAccess.DEFAULT, new URI(loginServer), clientId, refreshToken, clientSecret);
+                final TokenEndpointResponse tr = OAuth2.refreshAuthToken(HttpAccess.DEFAULT, new URI(loginServer), clientId, refreshToken, clientSecret,addlParamsMap);
```

Comment:
Are the additional parameters needed for refreshing the token or just when initiating login? If we only need them for login, then I don't think we need to change the AuthenticatorService.